pkgs/by-name/pr/project_lemonlime/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libsForQt5,
+  qt5,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "project_lemonlime";
```

```suggestion
  pname = "project-lemonlime";
```

As per the naming guidelines, hyphens should be used instead of underscores.